File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/CommandsModule.cs
using MVVMDemo.Commands;
using DevExpress.Xpf.DemoBase;

namespace MVVMDemo {
    public partial class CommandsModule : MVVMDemoModule {
        protected override IShowcaseInfo[] CreateShowcases() {
            const string path = "Modules/Commands";
            const string uri = "17353/Common-Concepts/MVVM-Framework/Commands/Delegate-Commands";
            const string viewModelBaseUri = "17351/Common-Concepts/MVVM-Framework/ViewModels/ViewModelBase";
            const string pocoUri = "17352/Common-Concepts/MVVM-Framework/ViewModels/POCO-ViewModels#";
            return new ShowcaseInfo[] {
                LoadShowcase("Delegate Commands", uri, path, new[] { typeof(DelegateCommandsViewModel), typeof(DelegateCommandsView) }),
                LoadShowcase("Delegate Commands Change Notifications", uri, path, new[] { typeof(ChangeNotificationsViewModel), typeof(ChangeNotificationsView) }),
                LoadShowcase("Automatic Commands Creation", viewModelBaseUri + "#Feature5", path, new[] { typeof(AutoCommandsViewModel), typeof(AutoCommandsView) }),
                LoadShowcase("Automatic Commands Change Notifications", viewModelBaseUri + "#Feature5", path, new[] { typeof(AutoCommandsChangeNotificationsViewModel), typeof(AutoCommandsChangeNotificationsView) }),
                LoadShowcase("POCO Commands", pocoUri + "Commands", path, new[] { typeof(POCOCommandsViewModel), typeof(POCOCommandsView), typeof(POCOCommandsViewModel_RuntimeGeneratedCode) }),
                LoadShowcase("POCO Commands Change Notifications", pocoUri + "Commands", path, new[] { typeof(POCOCommandsChangeNotificationsViewModel), typeof(POCOCommandsChangeNotificationsView), typeof(POCOCommandsChangeNotificationsViewModel_RuntimeGeneratedCode) }),
            };
        }
    }
}

File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/CommandsModule.vb
Imports MVVMDemo.Commands
Imports DevExpress.Xpf.DemoBase

Namespace MVVMDemo

    Public Partial Class CommandsModule
        Inherits MVVMDemoModule

        Protected Overrides Function CreateShowcases() As IShowcaseInfo()
            Const path As String = "Modules/Commands"
            Const uri As String = "17353/Common-Concepts/MVVM-Framework/Commands/Delegate-Commands"
            Const viewModelBaseUri As String = "17351/Common-Concepts/MVVM-Framework/ViewModels/ViewModelBase"
            Const pocoUri As String = "17352/Common-Concepts/MVVM-Framework/ViewModels/POCO-ViewModels#"
            Return New ShowcaseInfo() {LoadShowcase("Delegate Commands", uri, path, {GetType(DelegateCommandsViewModel), GetType(DelegateCommandsView)}), LoadShowcase("Delegate Commands Change Notifications", uri, path, {GetType(ChangeNotificationsViewModel), GetType(ChangeNotificationsView)}), LoadShowcase("Automatic Commands Creation", viewModelBaseUri & "#Feature5", path, {GetType(AutoCommandsViewModel), GetType(AutoCommandsView)}), LoadShowcase("Automatic Commands Change Notifications", viewModelBaseUri & "#Feature5", path, {GetType(AutoCommandsChangeNotificationsViewModel), GetType(AutoCommandsChangeNotificationsView)}), LoadShowcase("POCO Commands", pocoUri & "Commands", path, {GetType(POCOCommandsViewModel), GetType(POCOCommandsView), GetType(POCOCommandsViewModel_RuntimeGeneratedCode)}), LoadShowcase("POCO Commands Change Notifications", pocoUri & "Commands", path, {GetType(POCOCommandsChangeNotificationsViewModel), GetType(POCOCommandsChangeNotificationsView), GetType(POCOCommandsChangeNotificationsViewModel_RuntimeGeneratedCode)})}
        End Function
    End Class
End Namespace
